doc/api/http.md

```diff
@@ -513,11 +513,16 @@ See [`request.socket`][]
 ### request.end([data[, encoding]][, callback])
 <!-- YAML
 added: v0.1.90
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/18780
+    description: This method now returns a reference to `http.ClientRequest`.
```

Comment:
Nit: how about using just `ClientRequest` to be consistent with the other comments?

Reply:
Done.